src/uct/ib/base/ib_iface.c

```diff
@@ -787,26 +829,37 @@ int uct_ib_iface_is_reachable_v2(const uct_iface_h tl_iface,
 
     if (!uct_iface_is_reachable_params_valid(
                 params, UCT_IFACE_IS_REACHABLE_FIELD_DEVICE_ADDR)) {
+        uct_iface_populate_info_str_buf(params, "invalid is_reachable params");
```

Comment:
Can be more specific here.

Reply:
in case of invalid params, not sure we want to return any info string